### ISSUE TITLE
Everything ok before I failed to upload images thanks to "EACCES: permission denied, mkdir..."

### ISSUE THREAD

**Issue by vistaing** [TryGhost/Ghost-CLI#445]
This time I try to migrate another blog of mine from 0.11.11 to 1.5.2 according to [guidelines](https://docs.ghost.org/docs/migrating-to-ghost-1-0-0) from official docs. 

I successfully started new blog, imported text and images, but I just cannot upload new images in the editor. The error message says: 

> EACCES: permission denied, mkdir '/var/www/ghost/content/images/2017/08'
Try Again

It seems that I met some kind of authority problem but I cannot figure out what it is, because I followed every step in the guideline, and I even [succeeded once](https://github.com/TryGhost/Ghost-CLI/issues/407) (same problem never happened to that one). 

I even tried as root, but the problem shows again. 

> Environment
Digitalocean blank server, running Ghost v1.5.2 on Ubuntu 16.04.

**Comment by vikaspotluri123** [comment]
Looks like an issue with the user running ghost and the ownership of the directory. Since this isn't an issue with the core (bug report / feature request), you should really use the [slack channel](https://slack.ghost.org/), as outlined by the [support documentation](https://github.com/TryGhost/Ghost/blob/master/.github/SUPPORT.md).

**Comment by ErisDS** [comment]
I thought that this was already covered by the documentation, however it was missing from the migration guide.

I have added a note to the [migration guide](https://docs.ghost.org/docs/migrating-to-ghost-1-0-0#section-4-copy-your-images) to cover fixing ownership of the `content` folder after copying images.

I also added some clarity to a similar note in the [knowledge base](https://docs.ghost.org/docs/cli-knowledge-base#section--content) and expanded on the [EACESS permission denied](https://docs.ghost.org/docs/troubleshooting#section-error-eacces-permission-denied) entry in our troubleshooting docs.

Hopefully this will both prevent other people from having the same problem when migrating, and help them fix it if they have already run into this.
